test/arrays.js

```diff
@@ -194,6 +194,62 @@ describe('Array Diff', function() {
     ];
     assert.deepEqual(diff(parent, theirs, mine), expected);
   });
+
```

Comment:
Possible additional tests (not sure whether they'd return as expected, could be stubbed out for future if we aren't concerned now):

- Swap order of objects in array
- What if array doesn't contain objects (or some objects and some primitive values)?